gen-certs.sh

```diff
@@ -161,4 +161,4 @@ set -e
 genca
 gencert "velum" "Velum" "$all_hostnames" "$(ip_addresses)"
 gencert "salt-api" "salt-api.infra.caasp.local" "" "127.0.0.1"
-gencert "ldap" "OpenLDAP" "$all_hostnames" "$(ip_addresses)"
+gencert "ldap" "ldap.infra.caasp.local" "" "127.0.0.1"
```

Comment:
The "127.0.0.1" here is making the cert valid for "127.0.0.1" - which is generally considered bad practice at best, and a security issue at worst :)

Reply:
oops, that was still remainder of testing